src/ASM/LR/ASMu2D.C

```diff
@@ -2510,17 +2572,14 @@ bool ASMu2D::refine (const LR::RefineData& prm,
 }
 
 
-void ASMu2D::generateBezierBasis()
+void ASMu2D::generateBezierBasis ()
 {
-  const int p1 = geo->order(0);
-  const int p2 = geo->order(1);
-  bezier_u = this->getBezierBasis(p1);
-  bezier_v = this->getBezierBasis(p2);
-
+  bezier_u = getBezierBasis(geo->order(0));
+  bezier_v = getBezierBasis(geo->order(1));
```

Comment:
Since this method was introduced in the first commit of this PR, the cosmetic changes above (l2515-2519) should be squashed into that commit instead, to reduce the size of this commit